### PR TITLE
Fixing display of facial points in 1toN and NtoN

### PR DESCRIPTION
**Pull request by zuzi-m**
fixed displaying of facial points:
- after registration, facial points on original models are transformed back to their original positions
- from now facial points are also visible on registered models

### CODE REVIEW

File: GUI/src/cz/fidentis/gui/NavigatorTopComponent.java [hunk 1]
```diff
@@ -480,4 +492,30 @@ public void clearSelectionIfNeeded(ProjectTopComponent project) {
...
+
+    /**
+     * Creates a copy of given list of FacialPoints reversely transformed by given
+     * transformations. This is needed when displaying original FacialPoints after
+     * models were already registered by the transformations.
+     * @param points facial points of a model
+     * @param trans transformations used to the model
+     * @return copy of facial points reversely transformed by transformations.
+     */
+    private List<FacialPoint> applyReverseICPTrans(List<FacialPoint> points, List<ICPTransformation> trans) {
```
Comment (ZueFe) [inline GUI/src/cz/fidentis/gui/NavigatorTopComponent.java]: Bolo by dobre to hodit do nejakej samostatnej triedy spolu s dalsimi reversami (pravdepodobne do Processing modulu)

File: GUI/src/cz/fidentis/gui/NavigatorTopComponent.java [hunk 2]
```diff
@@ -480,4 +492,30 @@ public void clearSelectionIfNeeded(ProjectTopComponent project) {
...
+        // make a copy of points and also store their positions
+        for(int i=0;i<points.size();i++) {
+            FacialPoint point = new FacialPoint(points.get(i));
+            
+            result.add(point);
+            positions.add(point.getPosition());
+        }
+        
+        // apply reverse transformations to positions of points
+        Icp.instance().reverseAllTransformations(trans, positions, true);
```
Comment (ZueFe) [inline GUI/src/cz/fidentis/gui/NavigatorTopComponent.java]: Posledny parameter urcite nebude by default true, vezmi hodnotu z datoveho modelu, malo by sa to vola fpScaling abo cosik poodobne

File: GUI/src/cz/fidentis/gui/NavigatorTopComponent.java (hunk 1 above)
Comment (zuzi-m) [reply GUI/src/cz/fidentis/gui/NavigatorTopComponent.java]: good point, akurat neviem kde to dat. nasiel som triedu `cz.fidentis.processing.registration.Registrator` kde by sa to asi celkom hodilo ale neni to nikde pouzite :D pridam to asi tam kde su tie ostatne metody co aplikuju transformacie, nech je to aspon pokope

Comment (ZueFe) [reply GUI/src/cz/fidentis/gui/NavigatorTopComponent.java]: nuz ako hovorim najlepsie by bolo tomu vsetkemu spravit novu triedu

**Review by ZueFe — APPROVED** [review]

**Pull request merged**